back/engines/commercial/bulk_import_ideas/spec/acceptance/project_custom_fields_spec.rb

```diff
@@ -37,5 +37,17 @@
         assert_status 200
       end
     end
+
+    context 'in a timeline project without a current phase' do
+      let(:project) { create(:project_with_two_past_ideation_phases) }
+      let(:custom_form) { create(:custom_form, participation_context: project.phases.first) }
```

Comment:
Isn't the custom form linked to the project in the case of ideation?

Reply:
Yes that is a good point. Will just make that change